src/Components/DetailComponents/Aside.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { detailData } from "../../MockData/DetailData";
+
+const AsideComponent = () => {
+  let [facilityName, setFacilityName] = useState("all");
+
+  useEffect(() => {
+    sessionStorage.setItem("facilityName", `all`);
```

Comment:
```suggestion
    sessionStorage.setItem("facilityName", "all");
```
템플릿 리터럴을 사용하지 않을거라면 백틱은 어색해보입니다

Reply:
수정 완료